src/studio/src/designer/backend/Controllers/DatamodelsController.cs

```diff
@@ -154,7 +171,25 @@ public ActionResult<IEnumerable<AltinnCoreFile>> GetDatamodels(string org, strin
             var schemaFiles = _schemaModelService.GetSchemaFiles(org, repository, developer);
 
             return Ok(schemaFiles);
-        }        
+        }
+
+        /// <summary>
+        /// Method that returns the JSON contents of a specific datamodel.
+        /// </summary>
+        /// <param name="org">the org owning the models repo</param>
+        /// <param name="repository">the model repos</param>
+        /// <param name="modelPath">The path to the file to get.</param>        
+        [Authorize]
+        [HttpGet]
+        [ProducesResponseType(200)]
+        [Route("/designer/api/{org}/{repository}/datamodels/{*modelPath}")]
+        public async Task<IActionResult> Get(string org, string repository, string modelPath)
```

Comment:
We can also be more specific about returned value like so:
```suggestion
        public async Task<ActionResult<string>> Get(string org, string repository, string modelPath)
```
This matters if we want to generate an Open API specification.